components/List/list.css

```diff
@@ -0,0 +1,21 @@
+.list-container {
+    padding: 1rem;
+    width: 20rem;
+    border: 2px solid var(--primary-color);
+}
+
+.list-item {
+    list-style: none;
+    padding: 0.5rem;
+    margin-top: 0.2rem;
+    font-size: 1.1em;
+}
+
+.list-item:hover {
+    cursor: pointer;
+    color: var(--primary-color);
+}
+
+input[type="checkbox"] {
+    margin-right: 1rem;
+}
```

Comment:
i am not sure if this is a good practise, correct me please even i will get to know. I feel you should use a class instead.

Reply:
this totally depends if you want to put it into the generic segment yeah then this works but here I believe you want it to happen for a particular element therefore you should use class here.

Reply:
Ok got it